MakingCoffee/src/MakingCoffeeEx1.java

```diff
@@ -0,0 +1,48 @@
+
+import java.util.Scanner;
+public class MakingCoffeeEx1 {
```

Comment:
@Astgh78 jan rename class name , as we discussed yesterday no numbers

Reply:
Done!